src/styles/content/tables/table-cell-inactive/styles.less

```diff
@@ -0,0 +1,131 @@
+& when (@table-cell-inactive-enabled) {
+  /* Table */
+
+  .table {
+
+    tr.inactive td,
+    td.inactive {
+      .element-text-style(table-cell-inactive);
```

Comment:
@MatApple I reduced it to the bare minimum in this commit (Rafa asked me to help out to keep the PR moving, I hope you don't mind): https://github.com/dcos/dcos-ui/pull/1480/commits/3b4b6bcb9feb23cae4ef0c15e59eac0091ddaf65.

We'll just remove it when it's released in CNVS, so we won't be maintaining it in both places.

@ashenden If you'd like you can copy the mixins and variables definitions from this commit: https://github.com/dcos/dcos-ui/pull/1480/commits/5526d450c2b04ea91987b84a90aa1e88930caa3b